core/src/main/resources/error/error-classes.json

```diff
@@ -951,6 +951,45 @@
       "3. set \"spark.sql.legacy.allowUntypedScalaUDF\" to \"true\" and use this API with caution"
     ]
   },
+  "INVALID_PERCENTAGES" : {
+    "message" : [
+      "The percentage(s) "
+    ],
+    "subClass" : {
```

Comment:
Please, move to the error class `DATATYPE_MISMATCH`

Reply:
OK